include/nbl/asset/IImage.h

```diff
@@ -555,15 +555,15 @@ class IImage : public IDescriptor
 						break;
 				}
 
-				auto minPt = core::vector3du32_SIMD(off.x,off.y,off.z);
+				auto minPt = hlsl::uint32_t3(off.x,off.y,off.z);
 				auto srcBlockDims = asset::getBlockDimensions(srcImage->getCreationParameters().format);
 				if (((minPt%srcBlockDims)!=zero).any())
```

Comment:
`.any()` is not a method of HLSL/GLM vector

Reply:
whenever you change a templated function, you need to instantiate it to check it still compiles (or better, add a unit test to some example)